app/src/androidTest/java/com/simplemobiletools/calculator/MoneyActivityTest.java

```diff
@@ -84,6 +95,25 @@ public void taxWithoutGeoLocTest(){
         checkResult("25.99");
     }
 
+    @Ignore
+    @Test
+    public void testConversionSelection() {
```

Comment:
Is this test not functional or something ?

Reply:
Works locally, fails on the CI.